samples/react-native/package.json

```diff
@@ -30,7 +30,7 @@
     "axios": "^0.18.0",
     "prop-types": "^15.6.0",
     "react": "^16.4.1",
-    "react-native": "^0.55.4"
+    "react-native": "^0.57.8"
```

Comment:
We're not ready for this upgrade yet. There are several breaking changes (particularly around dev tooling) between 0.55.x and 0.56 and iirc 0.56 wasn't able to build on Windows machines. I believe the build problem was fixed in 0.57 but updating would still require some extra effort getting the `sitecore-jss-react-native` package and RN sample aligned with latest RN.

Reply:
Reverted as we discussed